app/src/vanilla/java/cm/aptoide/pt/app/view/EditorialItemsAdapter.java

```diff
@@ -33,25 +36,44 @@ public EditorialItemsAdapter(List<EditorialContent> editorialItemList,
 
   @Override
   public void onBindViewHolder(EditorialItemsViewHolder editorialItemsViewHolder, int position) {
+    if (editorialItemList.get(position)
+        .isPlaceHolderType()) {
+      this.appCardPlaceholderEditorialViewHolder = editorialItemsViewHolder;
```

Comment:
As I said on the before, i don't really agree with this way of storing the visibility